src/services/rickAndMortyApi.js

```diff
@@ -0,0 +1,12 @@
+export const getCharacters = page => {
+  return fetch(`https://rickandmortyapi.com/api/character?page=${page}`, {
+    headers: {
+      origin: null
+    }
+  })
+    .then(res => res.join())
```

Comment:
```suggestion
    .then(res => res.json())
```
I think you'll want `res.json()` in here, `res.join()` won't really do anything.